.github/workflows/continuous-integration.yml

```diff
@@ -7,8 +7,8 @@ on:
     - cron: "0 0 1 * *"
 
 jobs:
-  phpunit:
-    name: "PHPUnit"
+  unit-tests:
```

Comment:
I'd name it `pest`, it's more specific

Reply:
_Too_ specific IMO 🙂 when the unit test framework changed, I had to change the name, tho the task remained the same.